cloudbridge/cloud/providers/gce/provider.py

```diff
@@ -142,7 +142,7 @@ def __init__(self, config):
             'gce_service_creds_file', os.environ.get('GCE_SERVICE_CREDS_FILE'))
         self.credentials_dict = self._get_config_value(
             'gce_service_creds_dict',
-            json.loads(os.environ.get('GCE_SERVICE_CREDS_DICT') or {}))
+            json.loads(os.environ.get('GCE_SERVICE_CREDS_DICT') or '{}'))
         # If 'gce_service_creds_dict' is not passed in from config and
```

Comment:
[os.getenv()](https://docs.python.org/2/library/os.html#os.getenv) allows for a default:

`json.loads(os.getenv('GCE_SERVICE_CREDS_DICT', '{}'))`

Reply:
Done.

Thanks for review!